systemtest/benchtest/main.go

```diff
@@ -121,6 +125,23 @@ func benchmarkFuncName(f BenchmarkFunc) (string, error) {
 	return name, nil
 }
 
+func getLimiter(epm int) *rate.Limiter {
+	if epm < 0 {
```

Comment:
Should we treat the negative and 0 value equally? Or is there a use case for 0 EPS/EPM?

Reply:
Hmm, a 0 limit would mean that the bucket is not getting any tokens and would error out after sending 1000 events (the min bucket size). I don't think 0 limit has a good practical use case, we can treat them same as negative. Will update the PR.